internal/service/common/utils/repository.go

```diff
@@ -151,9 +151,10 @@ func Exists[T db.Model](ctx context.Context, db *pgxpool.Pool, uuid uuid.UUID) (
 // is returned.
 func Create[T db.Model](ctx context.Context, db *pgxpool.Pool, record T) (*T, error) {
 	nonNilTags := GetNonNilDBTagsFromStruct(record)
+	tags := GetAllDBTagsFromStruct(record)
 
 	// Return all columns to get any defaulted values that the DB may set
-	query := psql.Insert(im.Into(record.TableName()), im.Returning("*"))
+	query := psql.Insert(im.Into(record.TableName()), im.Returning(tags.Columns()...))
```

Comment:
Putting this back in because it makes the code too fragile.  If we add columns to the table, but forget to add them to the model it causes an error.   Until we have a better unit test strategy we should avoid making this more fragile than necessary.